<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrashDiContentSetting</class>
 <widget class="QWidget" name="CrashDiContentSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crash Swtich</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,2,2,1,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_prompt">
       <property name="text">
        <string>Guide tip:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_direction">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_0">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="hLayout_0" stretch="1,2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>func:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="customComboBox" name="comboBox_func"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="hLayout_1" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="customComboBox" name="comboBox_type"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="hLayout_6" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>inverse:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_inverse_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_directOther">
        <property name="text">
         <string>Boot, if you want to modify the relevant properties, please click on the model and then set in the right property window, other models are the same.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_warningToolTip">
       <property name="text">
        <string>Note: no interruptions during configuration</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ptb_previous">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ptb_next">
       <property name="text">
        <string>  Next  </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customComboBox</class>
   <extends>QComboBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
